docs/contributing.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: doc
+title: Contributing
+order: 60
+---
+
+# Contributing to the Documentation
+
+There are several ways you can contribute to improving the Dovecot
+documentation.
+
+## Submitting Fixes and Improvements
+
+If you are familiar with Git and [GitHub](https://github.com/), you can
+submit your changes directly to the documentation repository.
+
+1. **Fork the repository:** Start by creating a fork of the official
+   [Dovecot documentation repository](https://github.com/dovecot/documentation/)
+   on GitHub.
+2. **Clone your fork:** Clone your newly created fork to your local machine:
+   ```shell
+   git clone https://github.com/YOUR_USERNAME/documentation.git
+   cd documentation
+   ```
+3. **Create a new branch:** Create a new branch for your changes. Choose a
+   descriptive name for your branch.
+   ```shell
+   git checkout -b my-awesome-fix
+   ```
+4. **Make your changes:** Edit the documentation files as needed.
+5. **Commit your changes:** Once you are happy with your changes, commit
+   them with a descriptive commit message. The commit message should be in
+   the format `docs: A brief summary of the change`.
+   ```shell
+   git commit -am "docs: Fix a typo in the installation guide"
```

Comment:
This example could have `installation:` as the prefix. Although maybe we shouldn't need to be too strict in documentation commit messages, since they're less likely to be read in the future than code commits, but still would be nice to at least try.